openkfw/TruBudget
# Add git-secrets-hooks, add allowed example id

## Changes
Add a pattern which checks for azure-ids
Allow one example id which can be used in the code
Add script which set the hookpath of each contributor when executing "startDev.sh"
Following hooks are installed which check the commit for azure-ids:
 - commit-msg
 - pre-commit
 - prepare-commit-msg

## Testing
1. execute setupGitSecrets.sh
2. check what happens if git-secrets is not installed
3. try to commit an example secret locally (90d948b7-d819-488e-9629-9467a9b3b59f)
4. try to commit the allowed example secret (located in .gitallowed file)
